packages/web3/src/connect-button/connect-button.tsx

```diff
@@ -84,7 +84,21 @@ export const ConnectButton: React.FC<ConnectButtonProps> = (props) => {
 
   const chainSelect = renderChainSelect();
 
-  const buttonInnerText = (
+  const buttonInnerText = avatar ? (
+    <Space className="flex-row justify-content-center">
+      {restProps?.icon ? (
+        <>
+          <div className={classNames(`${prefixCls}-text`, hashId)}>{buttonText}</div>
+          <Avatar {...avatar} size="small" />
+        </>
+      ) : (
+        <>
+          <Avatar {...avatar} size="small" />
```

Comment:
It seems that the content is not centered. I tried to fix it, and it can be resolved by removing the top and bottom padding of the ant-btn, or you can see if there is a better solution.

<img width="302" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/45ce5308-36a2-46d9-bf15-f4b21eba9c58">

Reply:
Not a fan of inline style but I had to use it in order to make the content align.
![Screenshot from 2023-12-14 13-02-05](https://github.com/ant-design/ant-design-web3/assets/55346450/f15a4925-9fd7-4b3f-9117-c8cf09afa45f)
![Screenshot from 2023-12-14 13-02-31](https://github.com/ant-design/ant-design-web3/assets/55346450/2402870a-3869-42c3-a8e9-21b706a30d6c)